api/src/test/resources/org/openmrs/include/standardTestDataset.xml

```diff
@@ -447,4 +447,14 @@
   <test_order order_id="7" number_of_repeats="2" />
   <test_order order_id="8" />
   <test_order order_id="9" />
+	<concept_reference_map concept_map_id="19" concept_id="4088" concept_reference_term_id="13" concept_map_type_id="2" creator="1" date_created="2004-08-12 00:00:00.0" uuid="2f1a6a81-5c8b-40da-9266-d3d5013d9150"/>
+	<concept_reference_map concept_map_id="20" concept_id="4089" concept_reference_term_id="13" concept_map_type_id="2" creator="1" date_created="2004-08-12 00:00:00.0" uuid="e0eded3f-6c30-47ce-a324-1b63e710e075"/>
+	<concept_reference_map concept_map_id="21" concept_id="4090" concept_reference_term_id="13" concept_map_type_id="2" creator="1" date_created="2004-08-12 00:00:00.0" uuid="ed90a959-6a74-43b7-a01a-4de280421aa6"/>
+	<concept concept_id="4088" retired="false" datatype_id="1" class_id="1" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1" date_changed="2006-06-09 14:57:33.0" uuid="bb951f1d-efa9-45f4-969a-f05b6a944dd7"/>
+	<concept concept_id="4089" retired="false" datatype_id="1" class_id="1" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1" date_changed="2006-06-09 14:57:33.0" uuid="a0da2417-59be-47e4-81e7-d89fd5fcb892"/>
+	<concept concept_id="4090" retired="false" datatype_id="1" class_id="1" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1" date_changed="2006-06-09 14:57:33.0" uuid="a334a29c-0267-474e-8426-df0040b1da57"/>
+	<concept_reference_range concept_reference_range_id="30" concept_id="4090" criteria="#set( $criteria = $patient.getAge() &gt;= 1 &amp;&amp; $patient.getAge() &lt;= 70 )$criteria" hi_absolute="150.0" hi_normal="118.0" low_normal="80" low_absolute="80" creator="1" date_created="2024-01-01 00:00:00.0" retired="0" uuid="239c1904-15ff-45e1-ac9d-d83afb637926"/>
+	<concept_reference_range concept_reference_range_id="31" concept_id="4088" criteria="#set( $criteria = $patient.getAge() &gt;= 1 &amp;&amp; $patient.getAge() &lt;= 3 )$criteria" hi_absolute="120.0" hi_normal="118.0" low_normal="80" low_absolute="60" creator="1" date_created="2024-01-01 00:00:00.0" retired="0" uuid="5269e035-c8e6-44d7-af47-7a25a6cf6c60"/>
+	<concept_reference_range concept_reference_range_id="32" concept_id="4089" criteria="#set( $criteria = $patient.getAge() &gt;= 1 &amp;&amp; $patient.getAge() &lt;= 10 )$criteria" hi_absolute="120.0" hi_normal="118.0" low_normal="80" low_absolute="60" creator="1" date_created="2024-01-01 00:00:00.0" retired="0" uuid="d8ad3ceb-00a4-4484-a1ff-700f7ac3d10d"/>
```

Comment:
The `#set` and `$criteria` should not be included here. These are internal implementation details that the evaluating utility class should take care of. It should be enough for some one to just have `$patient.getAge() &gt;= 1 &amp;&amp; $patient.getAge() &lt;= 10` or `$patient.getAge() >= 1 && $patient.getAge() <= 10` in the user interface which is not in an xml document.